src/platform/forms-system/test/js/containers/FormPage.unit.spec.jsx

```diff
@@ -126,12 +131,12 @@ describe('Schemaform <FormPage>', () => {
     it('submit', () => {
       tree.getMountedInstance().onSubmit({});
 
-      expect(router.push.calledWith('next-page'));
+      expect(router.push.calledWith('/next-page')).to.be.true;
```

Comment:
I'm glad you got me to go back and work on this more... I couldn't figure out why all these test kept passing after I modified them 😭

Reply:
Great catch!